packages/block-library/src/block/edit.js

```diff
@@ -85,6 +98,31 @@ export default function ReusableBlockEdit( { attributes: { ref }, clientId } ) {
 		}
 	);
 
+	// local states for gudie modal
+	const [ isGudieOpen, setIsGudieOpen ] = useState( false );
+
+	const { saveEditedEntityRecord } = useDispatch( coreStore );
+	const { createNotice } = useDispatch( noticesStore );
+
+	// save the unsaved records
+	const saveEditedRecords = () => {
+		saveEditedEntityRecord( 'postType', 'wp_block', ref );
+		showSnackbar();
+	};
+
+	const showSnackbar = () => {
+		createNotice( 'success', __( 'Reusable block saved.' ), {
+			type: 'snackbar',
+			isDismissible: true,
+			actions: [
+				{
+					label: __( 'Learn more' ),
+					onClick: () => setIsGudieOpen( true ),
```

Comment:
This more of a design question (cc @critterverse), but I feel like the snackbar notice is a very brief interaction for it to be the only place to show a guide for reusable blocks.

Saving a reusable block is also towards the end of the process. I wonder if there's another more easily accessed place we'd also consider showing this towards the start of the flow of creating a reusable block. The modal where a user names the reusable block was one place I considered, but that would mean having two modals open. The block inspector for the reusable block is another option.

Anyway, this is not something to address in this PR, @thisissandip, so don't worry about it right now. 😄